django/core/management/__init__.py

```diff
@@ -302,7 +302,17 @@ def execute(self):
                 settings.configure()
 
         if settings.configured:
-            django.setup()
+            # Start the auto-reloading dev server even if the code is broken.
+            if subcommand == 'runserver' and '--noreload' not in self.argv:
```

Comment:
Could we abstract this into a method/property on the runserver command to make it a bit cleaner?